gen/drupal_elixir

```diff
@@ -104,6 +106,9 @@ foreach my $uid (sort keys %$membershipStruc) {
 }
 close (FILE_MEMBERSHIPS) or die "Cannot close $fileMemberships: $! \n";
 
+#duplicity in email is not allowed state
+if($errorInDuplicity) { die "ERROR: There are some duplicities in emails:\n" . $errorInDuplicity; }
```

Comment:
Unless you need printed data in files for some check, we can probably fail this right away on line 71, since check on duplicity is already done when processing groups->users->memberships.

Reply:
Or it can die right in the processUsers sub when the duplicity occurs. It is not necessary to continue to another user, because the script will die anyway.

Reply:
Moved.